packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordModal.tsx

```diff
@@ -97,16 +92,21 @@ const DxtradePasswordModal = () => {
     }, [accountType, hide, history, isDxtradePasswordNotSet, isLoading, isSuccess, onSubmit, password]);
 
     const successComponent = useMemo(() => {
+        const renderSuccessDescription = () => {
+            return isDemo
+                ? `Congratulations, you have successfully created your demo Deriv X account.`
```

Comment:
Lets not hardcode the platform names. Pls use the PlatformDetails constant

Reply:
good catch